addons/website/models/mixins.py

```diff
@@ -45,7 +45,8 @@ def _default_website_meta(self):
         """
         self.ensure_one()
         company = request.website.company_id.sudo()
-        title = (request.website or company).name
+        title = request.website.name
+        site_name = title
```

Comment:
Since there is no fallback on company anymore, this variable is now useless, just call `request.website.name` below